addons/website/static/src/core/interaction.js

```diff
@@ -47,6 +47,10 @@ export class Interaction {
         this.__colibri__.addDomListener(nodes, event, fn, options);
     }
 
+    registerCleanup(fn) {
+        this.__colibri__.cleanups.push(fn);
+    }
```

Comment:
What about cleanups that might become invalid earlier ? Should we avoid registering them, should we have a way to de-register them ? Or should all cleanups be implemented in a way that handles the fact that they might have already been run ?